chess.py

```diff
@@ -495,36 +509,38 @@ def __init__(self, master, board, remain=True):
                 piece_label = tk.Label(self, text=piece.APPEARANCE)
                 piece_label.grid(row=(piece_num + 2) * i + j + 1, column=0)
 
-                count_label = tk.Label(self, textvariable=self.counter[clr][piece][1])
+                count_label = tk.Label(self, textvariable=self.counter[clr][piece].s)
                 count_label.grid(row=(piece_num + 2) * i + j + 1, column=1)
 
         self.rowconfigure(piece_num + 1, weight=1)
 
-        self.load_vals()
+        self.reset()
 
     def reset(self):
         for clr in COLOURS:
             for piece in Piece.pieces:
-                self.counter[clr][piece][0] = 0
+                self.counter[clr][piece].i = 0
 
-        if self.remain:
+        if self.mode == "remaining":
             for tile in self.board.board.flat:
                 if tile.piece:
                     p = tile.piece
-                    self.counter[p.colour][p.__class__][0] += 1
+                    self.counter[p.colour][p.__class__].i += 1
+
+        elif self.mode != "taken":
```

Comment:
verplaats het raisen van deze error naar KillCounter.init
misschien mooier om dit elif self.mode == "taken": ...; else raise ValueError te maken.
zorg iig dat de ValueError iets zegt in de trant van f"Incorrect mode keyword: {self.mode}"
(om dit hele probleem te voorkomen kun je ook een enum aanmaken van mogelijke waardes maar dat is overkill)